recurly/client.py

```diff
@@ -31,50 +31,62 @@ def list_sites(self, **kwargs):
               results correspond to your request.
             * Records are returned in an arbitrary order. Since results are all
               returned at once you can sort the records yourself.
-        limit : int
-            Limit number of records 1-200.
-        order : str
-            Sort order.
-        sort : str
-            Sort field. You *really* only want to sort by `updated_at` in ascending
+        params.limit : int
```

Comment:
Trying this format so we can document the individual k/v pairs. It's not explicitly supported by sphinx but it works visually.